ios/brave-ios/App/l10n/tools/README.md

```diff
@@ -11,7 +11,18 @@ cd l10n/
 TOKEN=<token> ./push-strings-to-crowdin.sh
 ```
 
-If there any issues pushing strings to Transifex then these issues are logged to ```output.log```
+If there any issues pushing strings to Crowdin then these issues are logged to ```output.log```
+
+# How to pull strings from Crowdin
+ 
+You can check if all translation is finished on Crowdin Dev Portal. Once it is finished, the translated xliff files can be downloaded and imported into the project from Terminal:
+
+```
+cd l10n/
+TOKEN=<token> ./pull-translations-from-crowdin.sh
+```
+
+If there any issues downloading from Crowdin or importing strings to the project then these issues are logged to ```output.log```
 
 ### Note
 The API token can be created from Crowdin Dev Portal 
```

Comment:
Cant comment on it but just below:

```# How to contribute translations via Transifex```
to
```# How to contribute translations via Crowdin```

Reply:
There are also other references to Transifex in the README, probably want those to be altered as well

Reply:
Should we remove content about Transifex in README? since we are moving to Crowdin. and public cannot contribute to translation via Crowdin.

Reply:
README is mostly for us, so yeah we can update it to reflect CrowdIn reqs now